src/commonTest/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandlerTestsCommon.kt

```diff
@@ -925,37 +933,91 @@ class IncomingPaymentHandlerTestsCommon : LightningTestSuite() {
     }
 
     @Test
-    fun `mpp success then additional HTLC`() = runSuspendTest {
+    fun `mpp success then HTLC local replay`() = runSuspendTest {
```

Comment:
nit: I would use a more high-level name for this test:
```suggestion
    fun `fulfill locally replayed htlcs`() = runSuspendTest {
```